tests/test_script_queue.py

```diff
@@ -348,6 +349,126 @@ def make_add_data(location, locationSalIndex=0):
         with self.assertRaises(salobj.AckError):
             await self.remote.cmd_showScript.set_start(salIndex=3579, timeout=STD_TIMEOUT)
 
+    async def check_add_log_level(self, log_level):
+        """Test script log level when adding a script to the script queue."""
+        await self.assert_next_queue(enabled=False, running=True)
+
+        await self.remote.cmd_enable.start(timeout=STD_TIMEOUT)
+        await self.assert_next_queue(enabled=True, running=True)
+
+        # pause the queue so we know what to expect of queue state
+        await self.remote.cmd_pause.start(timeout=START_TIMEOUT)
+        await self.assert_next_queue(running=False)
+
+        async with salobj.Remote(domain=self.queue.domain, name="Script", index=I0) as script_remote:
```

Comment:
I know this was pre-existing, but I feel like `I0` should be `index0`, for clarity.

Reply:
I agree (though it is a global constant so INDEX0), but I think I0 makes incremented values much easier to understand by allowing me to focus on the increment, not the initial value. In other words I strongly prefer this:

    stop_data = self.make_stop_data([I0+6, I0+5, I0+4, I0, 5432], terminate=False)

to this:

    stop_data = self.make_stop_data([INDEX0+6, INDEX0+5, INDEX0+4, INDEX0, 5432], terminate=False)

Reply:
That's both fair, and unfortunate (the alternate usecase). I don't feel that strongly about it.